osquery/database/database.cpp

```diff
@@ -724,4 +817,67 @@ void dumpDatabase() {
     }
   }
 }
+
+Status serializeQueryDataRJ(const QueryData& q, rj::Document& d) {
+  if (!d.IsArray()) {
+    return Status(1, "Document is not an array");
+  }
+  for (const auto& r : q) {
+    rj::Document serialized;
+    serialized.SetObject();
+    auto s = serializeRowRJ(r, serialized);
```

Comment:
Just an FYI, there's an inconsistent use of `s` and `status` for the local `Status` variables.